src/Workspaces/CSharp/Portable/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -29,6 +29,12 @@ public static bool CanRemoveParentheses(this ParenthesizedExpressionSyntax node,
                 return true;
             }
 
+            // ((x, y)) -> (x, y)
+            if (expression.IsKind(SyntaxKind.TupleExpression))
```

Comment:
is this true always?  there's no worry that this might introduce an ambiguity in some cases?  in other words, do we need to check the parent at all?

Reply:
I think so. I couldn't come up with any scenario where it is useful to have parens around a tuple.

Reply:
Thanks!  LGTM then.

Reply:
So long as it is not a cast, this is legal, right?

Reply:
@DustinCampbell Right. Here we're dealing with a parenthesized expression, not a cast syntax.